roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -66,17 +66,20 @@ server {
     include fastcgi_params;
     fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
     fastcgi_param DOCUMENT_ROOT $realpath_root;
+    {% if ssl_terminated -%}
+    fastcgi_param HTTPS on;
+    {% endif %}
     fastcgi_pass unix:/var/run/php-fpm-wordpress.sock;
   }
 }
 
-{% if ssl_enabled %}
+{% if ssl_enabled or ssl_terminated %}
```

Comment:
The DRYist in me is like "nah" because most of it is duplicated, but extra verbosity doesn't usually hurt anyone. @fullyint what's your feeling on this?